### PR TITLE
At monday france

### PR DESCRIPTION
**Pull request by AudreyThomasson**

### CODE REVIEW

**Review by marceljones — APPROVED** [review]
Looking good!

File: pages/france.html [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        </header>
+        <div id="countryImages">
+            <img class="countryImages" src="/images/eiffel_tower.jpg" alt="Eiffel Tower">
+            <img class="countryImages" src="/images/ArcTriomphe.jpg" alt="Arc de Triomphe">
+        
+            <img class="countryImages" src="/images/versailles.jpg" alt="Versailles">
+            <img class="countryImages" src="/images/louvre.jpg" alt="Louvre">
+        </div>
+        <div>
+            <h1 style "color: red;">France</h1>
```
Comment (ste163) [inline pages/france.html]: need to remove the style "color: red;"

This is causing an issue with the HTML validator.

**Review by ste163 — APPROVED** [review]
Looks great!

**Review by tristan0508 — APPROVED** [review]
Looks great!

**Pull request merged**